alembic/operations/ops.py

```diff
@@ -1108,7 +1114,10 @@ def from_table(cls, table, _namespace_metadata=None):
         return cls(
             table.name,
             schema=table.schema,
-            table_kw=table.kwargs,
+            comment=table.comment,
+            info=table.info,
+            prefixes=table._prefixes,
```

Comment:
Given that this is a list and is hence mutable. shouldn't we copy it ? Same for info, which is a dict.

Reply:
I guess this is a good idea, seems to not be critical but then i may not be envisioning how these structures may be used, so sure.